### ISSUE TITLE
[VT-131] Adds ability to start/stop a task in vit - patch attached.

### ISSUE THREAD

**Issue by taskwarrior** [scottkosty/vit#152]
_dhananjay on 2017-02-11T13:48:49Z says:_ 

 Adds functionality to start/stop a task from vit, key is bound to 'b'
[I'm not sure this is the way to send in patches, but sadly couldn't find any documentation]

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2017-02-11T13:48:49Z
Modified: 2017-03-12T22:55:46Z
```

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2017-02-11T17:00:53Z says:_ 

 Thanks for the patch, dhananjay! The hardest part is deciding which key to use to bind. Is there a logical reason you chose 'b' ?

As for how to submit a patch, the most important thing is that the patch be with respect to the development branch (currently 1.3). Posting the patch here is fine. I slightly prefer Github because it has tools for reviewing patches: https://github.com/scottkosty/vit . But I would not want you to make an account just for this.

Let's see if [~djp] has an opinion. I think there are two questions here:
1. what should the keybinding(s) be? Before, we had discussed +/- to start/stop, but now I think that these should be reserved for tag-related functionality (someone is working on this by the way).

2. do we need two keybindings or is one keybinding for toggling enough? Before, we had discussed two. But now that I think about it, I'm not sure.

**Comment by taskwarrior** [comment]
_David Patrick on 2017-02-11T23:36:02Z says:_ 

 Personally, I use "\+" and "-" to start/ stop vit, but I can see an argument for a) a single toggle keybinding, and b) that the "+" might be applicable to some tag-related function. (I use "&" to trigger the tags report)

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2017-02-12T01:49:15Z says:_ 

 Thanks for your thoughts, DJP.

**Comment by taskwarrior** [comment]
_dhananjay on 2017-02-12T11:21:06Z says:_ 

 I chose 'b' for begin. But I'm not concerned about the key as such. I can send this over via github if you prefer.

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2017-02-12T19:47:01Z says:_ 

 If it's not too much trouble, then a github pull request would be ideal. If it would take time, then don't worry about it.

**Comment by taskwarrior** [comment]
_dhananjay on 2017-02-12T21:44:46Z says:_ 

 (at)scottk There you go https://github.com/scottkosty/vit/pull/6

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2017-02-13T01:39:43Z says:_ 

 Perfect, thanks!

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2017-03-12T22:55:46Z says:_ 

 The patch by Dhananjay will be included in 1.3 (note that keybinding is subject to change, depending on feedback). For the patch(es), see commit 2e53fa53 and follow-up commits.

**Comment by pbeckingham** [comment]
[VT-131_0001-Add-start-stop-commands-to-Vit.patch.txt](https://github.com/scottkosty/vit/files/1755271/VT-131_0001-Add-start-stop-commands-to-Vit.patch.txt)
